apps/condo/domains/marketplace/components/Invoice/UpdateInvoiceForm.tsx

```diff
@@ -0,0 +1,89 @@
+import { Invoice as InvoiceType, UserTypeType } from '@app/condo/schema'
+import get from 'lodash/get'
+import React, { useCallback, useMemo } from 'react'
+
+import { useMutation } from '@open-condo/next/apollo'
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+
+import { REGISTER_MULTI_PAYMENT_MUTATION } from '@condo/domains/acquiring/gql'
+import { runMutation } from '@condo/domains/common/utils/mutations.utils'
+import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
+import {
+    INVOICE_STATUS_PUBLISHED,
+    INVOICE_PAYMENT_TYPE_ONLINE,
+} from '@condo/domains/marketplace/constants'
+import { Invoice, InvoiceContext } from '@condo/domains/marketplace/utils/clientSchema'
+
+import { BaseInvoiceForm } from './BaseInvoiceForm'
+import { getPaymentLinkNotification } from './PaymentLinkNotification'
+
+
+type UpdateInvoiceFormProps = {
+    invoice: InvoiceType
+}
+
+export const UpdateInvoiceForm: React.FC<UpdateInvoiceFormProps> = ({ invoice }) => {
+    const intl = useIntl()
+    const { organization, link } = useOrganization()
+
+    const { obj: invoiceContext } = InvoiceContext.useObject({
+        where: {
+            organization: { id: organization.id },
+        },
+    })
+
+    const [registerMultiPayment] = useMutation(REGISTER_MULTI_PAYMENT_MUTATION)
+
+    const updateInvoiceAction = Invoice.useUpdate({}, async () => {
+        // replace after create marketplace/index page
+        // await router.push('/ticket')
```

Comment:
Already created
```suggestion
        await router.push('/marketplace?tab=bills')
```

Reply:
submit update form should return to invoice card, in create form fix it